src/Entity/Skills.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\SkillsRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass=SkillsRepository::class)
+ */
+class Skills
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private int $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private string $name;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
```

Comment:
There should be Constraints here for the form validator to do its job

Reply:
it's ok :)